crates/trie/sparse/src/trie.rs

```diff
@@ -769,15 +778,15 @@ impl<P> RevealedSparseTrie<P> {
         prefix_set: &mut PrefixSet,
         buffers: &mut RlpNodeBuffers,
     ) -> RlpNode {
-        let starting_path = buffers.path_stack.last().map(|item| item.path.clone());
+        let _starting_path = buffers.path_stack.last().map(|item| item.path.clone());
```

Comment:
this will still perform a clone if tracing is disabled, let's inline this in the `trace` macro itself

Reply:
Would it give the same behaviour? Since this takes the last item and its called in a method that mutates the path_stack:

```rust
        'main: while let Some(RlpNodePathStackItem { level, path, mut is_in_prefix_set }) =
            buffers.path_stack.pop()
        {
            let node = self.nodes.get_mut(&path).unwrap();
            trace!(
                target: "trie::sparse",
                ?_starting_path,
                ?level,
                ?path,
                ?is_in_prefix_set,
                ?node,
                "Popped node from path stack"
            );
   ```